js/src/components/SnippetPreviewSection.js

```diff
@@ -17,6 +17,22 @@ const Section = styled( StyledSection )`
 	}
 `;
 
+/**
+ * Function to map the editor data to data for the preview.
+ *
+ * @param {Object} mappedData The default mappedData object.
+ * @param {string} mappedData.title The processed title.
+ * @param {string} mappedData.url The baseUrl with the slug.
+ * @param {string} mappedData.description The processed description.
+ *
+ * @returns {Object} The new mappedData object.
+ */
+const mapDataToPreview = function( mappedData ) {
```

Comment:
Here it's also confusing that you're mapping data that already has been mapped according to the argument name